lindera-tantivy 0.19 doesn't compile

There's a compile error due to invalid arguments
```
➜  lindera-tantivy git:(main) cargo build
    Blocking waiting for file lock on build directory
   Compiling lindera-tantivy v0.19.0 (/home/pascal/LinuxData/Development/lindera-tantivy)
error[E0061]: this function takes 4 arguments but 0 arguments were supplied
   --> src/tokenizer.rs:34:24
    |
34  |             tokenizer: LTokenizer::new()?,
    |                        ^^^^^^^^^^^^^^^-- multiple arguments are missing
    |
note: associated function defined here
   --> /home/pascal/.cargo/registry/src/github.com-1ecc6299db9ec823/lindera-0.19.5/src/tokenizer.rs:241:12
    |
241 |     pub fn new(
    |            ^^^
help: provide the arguments
    |
34  |             tokenizer: LTokenizer::new(/* lindera_core::dictionary::Dictionary */, /* Option<lindera_core::user_dictionary::UserDictionary> */, /* lindera_core::viterbi::Mode */, /* bool */)?,
    |                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

error[E0277]: the `?` operator can only be applied to values that implement `Try`
  --> src/tokenizer.rs:34:24
   |
34 |             tokenizer: LTokenizer::new()?,
   |                        ^^^^^^^^^^^^^^^^^^ the `?` operator cannot be applied to type `lindera::tokenizer::Tokenizer`
   |
   = help: the trait `Try` is not implemented for `lindera::tokenizer::Tokenizer`

error[E0599]: no function or associated item named `with_config` found for struct `lindera::tokenizer::Tokenizer` in the current scope
  --> src/tokenizer.rs:40:36
   |
40 |             tokenizer: LTokenizer::with_config(config)?,
   |                                    ^^^^^^^^^^^ function or associated item not found in `lindera::tokenizer::Tokenizer`
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "lindera-tantivy"
-version = "0.19.0"
+version = "0.19.1"
 edition = "2021"
 description = "Lindera Tokenizer for Tantivy."
 documentation = "https://docs.rs/lindera-tantivy"
@@ -24,7 +24,7 @@ compress = ["lindera/compress"]
 [dependencies]
 tantivy = "0.19.0"
 
-lindera = "0.19.0"
+lindera = "0.19.5"
 
 [dev-dependencies]
 criterion = { version = "0.4.0", features = ["html_reports"] }

--- a/README.md
+++ b/README.md
@@ -89,12 +89,13 @@ fn main() -> tantivy::Result<()> {
         dictionary,
         user_dictionary: None,
         mode: Mode::Normal,
+        with_details: false,
     };
 
     // register Lindera tokenizer
     index
         .tokenizers()
-        .register("lang_ja", LinderaTokenizer::with_config(config).unwrap());
+        .register("lang_ja", LinderaTokenizer::from_config(config).unwrap());
 
     // create index writer
     let mut index_writer = index.writer(50_000_000)?;

--- a/benches/bench.rs
+++ b/benches/bench.rs
@@ -53,6 +53,7 @@ fn bench_indexing(c: &mut Criterion) {
         dictionary,
         user_dictionary: None,
         mode: Mode::Normal,
+        with_details: false,
     };
 
     // Test document set.
@@ -68,7 +69,7 @@ fn bench_indexing(c: &mut Criterion) {
     // register Lindera tokenizer
     index
         .tokenizers()
-        .register("lang_ja", LinderaTokenizer::with_config(config).unwrap());
+        .register("lang_ja", LinderaTokenizer::from_config(config).unwrap());
 
     // create index writer
     let mut index_writer = index.writer(50_000_000).unwrap();

--- a/examples/cc-cedict_example.rs
+++ b/examples/cc-cedict_example.rs
@@ -65,12 +65,13 @@ fn main() -> tantivy::Result<()> {
         dictionary,
         user_dictionary: None,
         mode: Mode::Normal,
+        with_details: false,
     };
 
     // register Lindera tokenizer
     index
         .tokenizers()
-        .register("lang_zh", LinderaTokenizer::with_config(config).unwrap());
+        .register("lang_zh", LinderaTokenizer::from_config(config).unwrap());
 
     // create index writer
     let mut index_writer = index.writer(50_000_000)?;

--- a/examples/ipadic_example.rs
+++ b/examples/ipadic_example.rs
@@ -65,12 +65,13 @@ fn main() -> tantivy::Result<()> {
         dictionary,
         user_dictionary: None,
         mode: Mode::Normal,
+        with_details: false,
     };
 
     // register Lindera tokenizer
     index
         .tokenizers()
-        .register("lang_ja", LinderaTokenizer::with_config(config).unwrap());
+        .register("lang_ja", LinderaTokenizer::from_config(config).unwrap());
 
     // create index writer
     let mut index_writer = index.writer(50_000_000)?;

--- a/examples/ko-dic_example.rs
+++ b/examples/ko-dic_example.rs
@@ -65,12 +65,13 @@ fn main() -> tantivy::Result<()> {
         dictionary,
         user_dictionary: None,
         mode: Mode::Normal,
+        with_details: false,
     };
 
     // register Lindera tokenizer
     index
         .tokenizers()
-        .register("lang_ko", LinderaTokenizer::with_config(config).unwrap());
+        .register("lang_ko", LinderaTokenizer::from_config(config).unwrap());
 
     // create index writer
     let mut index_writer = index.writer(50_000_000)?;

--- a/examples/unidic_example.rs
+++ b/examples/unidic_example.rs
@@ -65,12 +65,13 @@ fn main() -> tantivy::Result<()> {
         dictionary,
         user_dictionary: None,
         mode: Mode::Normal,
+        with_details: false,
     };
 
     // register Lindera tokenizer
     index
         .tokenizers()
-        .register("lang_ja", LinderaTokenizer::with_config(config).unwrap());
+        .register("lang_ja", LinderaTokenizer::from_config(config).unwrap());
 
     // create index writer
     let mut index_writer = index.writer(50_000_000)?;

--- a/src/tokenizer.rs
+++ b/src/tokenizer.rs
@@ -1,6 +1,7 @@
 use tantivy::tokenizer::{BoxTokenStream, Tokenizer as TTokenizer};
 
 use lindera::{
+    builder,
     tokenizer::{
         DictionaryConfig as LDictionaryConfig, Tokenizer as LTokenizer,
         TokenizerConfig as LTokenizerConfig,
@@ -8,8 +9,8 @@ use lindera::{
     DictionaryKind as LDictionaryKind, Token as LToken,
 };
 
-use crate::stream::LinderaTokenStream;
 use crate::LinderaResult;
+use crate::{mode::Mode, stream::LinderaTokenStream};
 
 pub type DictionaryConfig = LDictionaryConfig;
 pub type DictionaryKind = LDictionaryKind;
@@ -30,14 +31,16 @@ impl Clone for LinderaTokenizer {
 
 impl LinderaTokenizer {
     pub fn new() -> LinderaResult<LinderaTokenizer> {
+        let dictionary = builder::load_dictionary_from_kind(DictionaryKind::IPADIC)?;
+
         Ok(LinderaTokenizer {
-            tokenizer: LTokenizer::new()?,
+            tokenizer: LTokenizer::new(dictionary, None, Mode::Normal, false),
         })
     }
 
-    pub fn with_config(config: TokenizerConfig) -> LinderaResult<LinderaTokenizer> {
+    pub fn from_config(config: TokenizerConfig) -> LinderaResult<LinderaTokenizer> {
         Ok(LinderaTokenizer {
-            tokenizer: LTokenizer::with_config(config)?,
+            tokenizer: LTokenizer::from_config(config)?,
         })
     }
 }
@@ -83,10 +86,11 @@ mod tests {
             dictionary,
             user_dictionary: None,
             mode: Mode::Normal,
+            with_details: false,
         };
 
         let tokens = test_helper(
-            LinderaTokenizer::with_config(config)
+            LinderaTokenizer::from_config(config)
                 .unwrap()
                 .token_stream("すもももももももものうち"),
         );

